util/cron/test-perf.cray-cs.ofi.bash

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# Run performance tests with comm=ofi on a cray-cs system.
+
+CWD=$(cd $(dirname $0) ; pwd)
+
+export CHPL_TEST_PERF_CONFIG_NAME='16-node-cs'
+
+source $CWD/common-perf.bash
```

Comment:
Need to set `export CHPL_TEST_PERF_DIR=/cray/css/users/chapelu/NightlyPerformance/cray-cs/16-node-cs`

Reply:
Will do, in a follow-up PR.